### PR TITLE
doc: matter: update tools page

### PR DESCRIPTION
**Pull request by greg-fer**
Edited Matter tools page with chip-cert and CHIP Tool. Added the chip-cert readme to Matter docset.
KRKNWK-16716.

----

- [x] Update after #10750 is merged
- [x] Update Matter upstream with pages for chip-cert and spake2p tools

### CODE REVIEW

**Comment by greg-fer** [comment]
@HubertHugo , @ArekBalysNordic , @doublemis1 , please have a look.

File: doc/nrf/protocols/matter/getting_started/tools.rst [hunk 1]
```diff
@@ -184,6 +204,56 @@ You can also install the ZAP tool using one of the following methods:
...
+SPAKE2+ Python tool
+*******************
+
+SPAKE2+ Python Tool is a Python script for generating SPAKE2+ protocol parameters.
+The protocol is used during Matter commissioning to :ref:`establish a secure session <ug_matter_overview_commissioning_stages_case>` between the commissioner and the commissionee.
+
+.. note::
+   Currently, the tool only supports generating Verifier parameters.
+
+For usage examples, see the `SPAKE2+ Python Tool page`_ in the Matter SDK official documentation.
```
Comment (greg-fer) [inline doc/nrf/protocols/matter/getting_started/tools.rst:253]: @Damian-Nordic , could you have a look at these Spake2+ lines?

Comment (Damian-Nordic) [reply doc/nrf/protocols/matter/getting_started/tools.rst:253]: LGTM

**Review by Damian-Nordic — APPROVED** [review]

File: doc/nrf/protocols/matter/getting_started/tools.rst [hunk 2]
```diff
@@ -184,6 +204,56 @@ You can also install the ZAP tool using one of the following methods:
 
 Both these methods still require adding the ZAP tool location to the system :envvar:`PATH` environment variables, as detailed in the step 3 above.
 
+.. _ug_matter_gs_tools_cert:
+
+CHIP Certificate Tool
+*********************
+
+Matter's CHIP Certificate Tool, in short chip-cert, is a command-line utility tool for generating and editing Matter certificates such as Certificate Declarations (CD), Device Attestation Certificates (DAC), Product Attestation Intermediate (PAI) certificate, alongside their related keys.
```
Comment (doublemis1) [inline doc/nrf/protocols/matter/getting_started/tools.rst:212]: If you want to list all available certificates which can be generated using chip-cert, then a PAA can also be generated using chip-cert

**Review by HubertHugo — APPROVED** [review]

**Pull request merged**